cpu/esp_common/esp-now/esp_now_params.h

```diff
@@ -30,42 +88,42 @@
  * @brief   The size of the stack used for the ESP-NOW netdev driver thread.
  * @ingroup cpu_esp_common_conf
  */
-#ifndef ESP_NOW_STACKSIZE
-#define ESP_NOW_STACKSIZE       (THREAD_STACKSIZE_DEFAULT)
+#ifndef CONFIG_ESP_NOW_STACKSIZE
```

Comment:
I decided to rename it for two reasons:

1. I wished the stack size would be configurable, but I couldn't find an usable approach til now. The option has to be an `int` which doesn't allow a default like `THREAD_STACKSIZE_DEFAULT`. Maybe, I will add a combination of a `bool` option for using the default and an `int` option for specifying a different value.

2. I wanted to avoid naming inconsistencies from the users point of view in documentations where you have tables like

   Configuration Parameter    | Description
   ------------------------------------|----------------
   #CONFIG_ESP_WIFI_SSID      | SSID of the AP to be used.
   #CONFIG_ESP_WIFI_PASS      | Passphrase used for the AP as clear text
   #CONFIG_ESP_WIFI_STACKSIZE | Stack size used for the WiFi netdev driver

   I wanted to avoid tables like the following:

   Configuration Parameter    | Description
   ------------------------------------|----------------
   #CONFIG_ESP_WIFI_SSID      | SSID of the AP to be used.
   #CONFIG_ESP_WIFI_PASS      | Passphrase used for the AP as clear text
   #ESP_WIFI_STACKSIZE | Stack size used for the WiFi netdev driver

  Therefore, I decided to name everything that is configurable by user also at command line in same way.

Reply:
I decided to rename it for two reasons:

1. I wished the stack size would be configurable, but I couldn't find an usable approach til now. The option has to be an `int` which doesn't allow a default like `THREAD_STACKSIZE_DEFAULT`. Maybe, I will add a combination of a `bool` option for using the default and an `int` option for specifying a different value.

2. I wanted to avoid naming inconsistencies from the users point of view in documentations where you have tables like

   Configuration Parameter    | Description
   ------------------------------------|----------------
   #CONFIG_ESP_WIFI_SSID      | SSID of the AP to be used.
   #CONFIG_ESP_WIFI_PASS      | Passphrase used for the AP as clear text
   #CONFIG_ESP_WIFI_STACKSIZE | Stack size used for the WiFi netdev driver

   I wanted to avoid tables like the following:

   Configuration Parameter    | Description
   ------------------------------------|----------------
   #CONFIG_ESP_WIFI_SSID      | SSID of the AP to be used.
   #CONFIG_ESP_WIFI_PASS      | Passphrase used for the AP as clear text
   #ESP_WIFI_STACKSIZE | Stack size used for the WiFi netdev driver

  Therefore, I decided to name everything that is configurable by user also at command line in same way.